queue_services/entity-emailer/src/entity_emailer/email_templates/common/continuation-application-details.html

```diff
@@ -0,0 +1,25 @@
+<div class="continuation-application-details">
+  <p class="title-message bold">Your Continuation Application Details</p>
+  <div>
+    <div class="name bold">Identifying Number in Foreign Jurisdiction:</div>
+    <div class="value">{{ filing.foreignJurisdiction.identifier }}</div>
+  </div>
+  [[16px.html]]
+  <div>
+    <div class="name bold">Name in Foreign Jurisdiction:</div>
+    <div class="value">{{ filing.foreignJurisdiction.legalName }}</div>
+  </div>
+  [[16px.html]]
+  <div>
+    <div class="name bold">Foreign Jurisdiction:</div>
+    <div class="value">{{ foreign_jurisdiction }}</div>
+  </div>
+
+  {% if filing.mode == 'EXPRO' %}
```

Comment:
cannot depend on mode as its not expected in schema (only used in UI), you can check if business object is exist. ref: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/report.py#L936

Reply:
Good catch. Fixed - see latest commit.

Expro application:
![image](https://github.com/user-attachments/assets/66d3b12e-6680-4136-b4eb-935ee0321a12)

Manual entry application:
![image](https://github.com/user-attachments/assets/641f285b-e11a-45f8-9088-459be1287c95)